secure/secure.sh

```diff
@@ -50,8 +50,11 @@ REALM=$(echo $DOMAIN | awk '{print toupper($0)}')
 MASTER="$(/usr/share/google/get_metadata_value attributes/dataproc-master)"
 MASTER_FQDN="$MASTER.$DOMAIN"
 HADOOP_CONF_DIR="/etc/hadoop/conf"
-HADOOP_VERSION=$(hadoop version | head -n 1)
 
+readonly cross_realm_trust_realm="$(/usr/share/google/get_metadata_value attributes/cross-realm-trust-realm)"
```

Comment:
I think this you need a default for these optional arguments (e.g. `echo ''`)

Reply:
Done.